src/background/sidebar-injector.js

```diff
@@ -239,6 +251,10 @@ export function SidebarInjector() {
       await injectIntoPDF(tab);
     } else if (type === CONTENT_TYPE_VITALSOURCE) {
       await injectIntoVitalSourceReader(tab, config);
+    } else if (type === CONTENT_TYPE_LMS) {
+      throw new BlockedSiteError(
+        "Hypothesis extension can't be used on learning platform assignments"
```

Comment:
```suggestion
        "Hypothesis extension can't be used on Hypothesis LMS assignments"
```

We need to be clear that this is specifically about Hypothesis LMS assignments. The user might not know what "LMS" means, but we do use the term extensively in our docs so it is easy to search for. Searching Google for "Hypothesis LMS" turns up https://web.hypothes.is/education/lms/.